packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.test.tsx

```diff
@@ -26,6 +26,6 @@ describe('TimePicker', () => {
       />
     );
 
-    expect(container.queryByLabelText(/timepicker open button/i)).toBeInTheDocument();
+    expect(container.queryByLabelText(/Time range editor/i)).toBeInTheDocument();
```

Comment:
Should we go with "Time range editor" or "Time range picker" here? Just want to make sure whatever it is, it's consistent.

Reply:
Good point, I'll change it to TimeRange picker instead